x/common/testutil/cli/network_test.go

```diff
@@ -1,20 +1,30 @@
-//go:build norace
-// +build norace
-
 package cli_test
 
+// Alteration of [network/network_test.go](https://github.com/cosmos/cosmos-sdk/blob/v0.45.15/testutil/network/network_test.go)
+//
+// ```go
+// //go:build norace
```

Comment:
Do we need these double-commented comments?

Reply:
Removed them. I didn't understand why the SDK team wanted this file ignored in the build anyway. It got deprecated in v0.47, so the `go:build norace` may have been zombie code without strong reasoning.